homework13/src/main/java/ua/lviv/home/dtos/BucketProductDto.java

```diff
@@ -0,0 +1,11 @@
+package ua.lviv.home.dtos;
+
+import ua.lviv.home.enteties.Product;
+
+import java.util.Date;
+
+public class BucketProductDto {
+    public Integer id;
```

Comment:
Better to use primitive types where possible.

Reply:
done)